src/IECoreImage/ClientDisplayDriver.cpp

```diff
@@ -33,17 +33,17 @@
 //
 //////////////////////////////////////////////////////////////////////////
 
+#include "boost/array.hpp"
+#include "boost/asio.hpp"
```

Comment:
If I understand correctly, this is purely about asio.hpp, right? If so, let's leave the other boost includes where they belong, and add a quick comment saying something like`// Include asio.hpp first to avoid conflicts on Windows`. Otherwise we're gonna wonder why we're not following the include convention, and might inadvertently break the Windows build by "tidying things up".

Reply:
This is breaking things on Mac - it looks like `asio.hpp` includes `intrusive_ptr.hpp`, and we need to include that first via `RefCounted.h` to apply our symbol visibility fixes to it. Is it sufficient for you to have `asio.hpp` second, after `IECoreImage/ClientDisplayDriver.h`?